research/templates/research/browse.html

```diff
@@ -291,6 +291,7 @@ <h5 class="card-title float-start">
             </div>
 
             <div class="modal-footer">
+                <button type="button" class="btn btn-primary dlg-multi-action-button hidden" data-action="" data-name="" data-collection="" data-overwrite=""><span class="action"></span></button>

```

Comment:
Can we move the 'Copy and overwrite all' button to the right of the close button? The copy button now jumps from right to left when overwrite action is necessary.